bentoml/saved_bundle/loader.py

```diff
@@ -177,6 +177,7 @@ def load(bundle_path):
 
     svc_cls = load_bento_service_class(bundle_path)
     svc = svc_cls()
+    svc._path = bundle_path
```

Comment:
I am not sure about this, there may be a better way of getting the bundle path.

Reply:
you can use `svc._bento_service_bundle_path`